doc/contributing.md

```diff
@@ -72,16 +243,32 @@ download and install anything:
 * Done!
 
 Alternatively, you can make the changes offline to the files in the `doc` folder or the
-example scripts. See [Contributing Code](#contributing-code) for instructions.
+example scripts. See [building the documentation](#building-the-documentation) for
+instructions.
+
+### Building the Documentation
```

Comment:
How about a subsection title like "Editing the Documentation Locally" or "Editing the Documentation Offline" or something else? It would be more consistent with "Editing the Documentation on GitHub" above.